metro.go

```diff
@@ -58,7 +58,20 @@ func (i *arrayFlags) Set(value string) error {
 	return nil
 }
 
-func initLogging(to_file bool, level string) {
+type Exit struct{ Code int }
+
+// exit code handler
+func handleExit() {
+	if e := recover(); e != nil {
+		if exit, ok := e.(Exit); ok == true {
+			os.Exit(exit.Code)
+		}
+		panic(e) // not an Exit, bubble up
+	}
+}
+
+func initLogging(to_file bool, level string) log.LoggerInterface {
+	defer handleExit()
```

Comment:
Do we need this `defer`? Wouldn't a `panic` here be already handled by the `defer handleExit()` in `main`?